unison-cli/src/Unison/CommandLine/OutputMessages.hs

```diff
@@ -2066,6 +2066,11 @@ notifyUser dir = \case
         <> if remote == local
           then P.group (prettyProjectAndBranchName remote <> ".")
           else prettyProjectAndBranchName remote <> "as" <> P.group (prettyProjectAndBranchName local <> ".")
+  RenamedProject oldName newName ->
+    pure . P.wrap $
+      if oldName == newName
+        then "Ok, I didn't rename" <> P.group (prettyProjectName oldName <> ".")
```

Comment:
```suggestion
        then prettyProjectName oldName <> "is already named" <> P.group (prettyProjectName oldName <> "!") <> "😄"
```

Reply:
I thought the original message was kind of confusing without the context of the successful variant.

Reply:
Done